sys/include/net/ng_netif.h

```diff
@@ -90,6 +90,41 @@ void ng_netif_remove(kernel_pid_t pid);
  */
 kernel_pid_t *ng_netif_get(size_t *size);
 
+/**
+ * @brief   Converts a hardware address to a human readable format.
```

Comment:
hm, ok. Just wanted to stress that the string is NULL terminated to be used in string functions (as `printf("%s", ...)`